src/lib/temple/beacon.test.ts

```diff
@@ -0,0 +1,80 @@
+// import { crypto_sign_ed25519_pk_to_curve25519, crypto_sign_ed25519_sk_to_curve25519 } from 'libsodium-wrappers';
+import { browser } from 'webextension-polyfill-ts';
+
+import {
+  // createCryptoBox,
+  fromHex,
+  getDAppPublicKey,
+  getOrCreateKeyPair,
+  removeDAppPublicKey,
+  saveDAppPublicKey,
+  toHex,
+  toPubKeyStorageKey
+} from './beacon';
+
+const SAMPLE_PAYLOAD = 'hello, world!';
+
+describe('Beacon tests', () => {
+  describe('Storage', () => {
```

Comment:
use complex tests structure that we are using in TW mobile (`shelter.spec.ts` for example)
```typescript
describe('File name', () => {
    describe('function1 name', () => {
        it('should .... do something', () => {});
        it('should .... do something', () => {});
        it('should .... do something', () => {});
    });

    describe('function2 name', () => {
        it('should .... do something', () => {});
        it('should .... do something', () => {});
        it('should .... do something', () => {});
    });
});
```